src/quo/components/animated_header.cljs

```diff
@@ -10,7 +10,7 @@
 
 (defn header-wrapper-style [{:keys [value offset]}]
   (merge
-   {:background-color :white}
+   {:background-color (:ui-background @colors/theme)}
```

Comment:
could you elaborate on why you're using deref here?

Reply:
I've made colors which are dynamic in Quo as an atom, so it would be as a reminder that this value is dynamic and should be used only in functions (called during render)